cdap-watchdog/src/main/java/co/cask/cdap/logging/save/LogSaver.java

```diff
@@ -67,6 +73,9 @@ public LogSaver(KafkaClientService kafkaClient,
     this.kafkaCancelMap = new HashMap<>();
     this.kafkaCancelCallbackLatchMap = new HashMap<>();
     this.messageProcessors = messageProcessors;
+
+    this.metricsContext = metricsCollectionService.getContext(
+      ImmutableMap.of(Constants.Metrics.Tag.COMPONENT, Constants.Service.LOGSAVER));
```

Comment:
we do not aggregate for instances for system services right now.. so it'll be bigger change. CDAP-3265